src/EditorFeatures/Core/Classification/Syntactic/SyntacticClassificationTaggerProvider.cs

```diff
@@ -17,52 +17,39 @@
 using Microsoft.VisualStudio.Text.Tagging;
 using Microsoft.VisualStudio.Utilities;
 
-namespace Microsoft.CodeAnalysis.Classification
+namespace Microsoft.CodeAnalysis.Classification;
```

Comment:
ouch. not expecting this to all be in the same commit (git blame will not be helpful for figuring out the change)